product_packaging_dimension/models/product_packaging.py

```diff
@@ -13,11 +13,6 @@ class ProductPackaging(models.Model):
         ("positive_height", "CHECK(height>=0)", "Height must be positive"),
         ("positive_width", "CHECK(width>=0)", "Width must be positive"),
         ("positive_length", "CHECK(packaging_length>=0)", "Length must be positive"),
-        (
-            "positive_max_weight",
-            "CHECK(max_weight>=0)",
-            "Max Weight must be positive",
```

Comment:
You could keep it but on weight instead of max_weight

Reply:
Done